source/pages/Transactions/TransactionConfirmation.tsx

```diff
@@ -152,6 +152,8 @@ const TransactionConfirmation: React.FC<ITransactionConfirmation> = ({
                     {typeof item.value === 'string' && item.value.length > 10
                       ? ellipsis(item.value)
                       : item.value}
+                    {typeof item.value === 'boolean' && `${item.value}`}
```

Comment:
```suggestion
                    {typeof item.value === 'boolean' && (item.value ? 'Yes' : 'No')}
```

we can display yes or no for boolean values, it's more human-readable